src/System.Threading.Tasks.Extensions/src/System.Threading.Tasks.Extensions.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You will also need to add a new .builds file if you want this to be built as part of the full product build, which i believe you do.